ts/components/services/ServicePreferenceSummary.tsx

```diff
@@ -20,12 +20,12 @@ export const getServicesPreferenceModeLabel = (
   mode: ServicesPreferencesModeEnum
 ): string =>
   ({
-    [ServicesPreferencesModeEnum.AUTO]: I18n.t(
+    [ServicesPreferencesModeEnum.AUTO]: `${I18n.t(
```

Comment:
This function is used in 2 places and this is the effect in the other part

<img src="https://user-images.githubusercontent.com/822471/124957976-f2f6d900-e019-11eb-8ec0-2104efa8b2f7.jpg" height="600"/>

My suggestion is to keep this function "private" (remove `export`) and make a new one in the other component `ts/screens/profile/PreferencesScreen.tsx`

Reply:
I didn't notice it was used on preferences screen too
0fbb5df fixes